src/main/scala/org/broadinstitute/dsde/workbench/sam/util/DatabaseSupport.scala

```diff
@@ -11,9 +12,12 @@ trait DatabaseSupport {
   protected val cs: ContextShift[IO]
   protected val dbRef: DbReference
 
-  protected def runInTransaction[A](databaseFunction: DBSession => A): IO[A] = {
-    cs.evalOn(ecForDatabaseIO)(IO {
-      dbRef.inLocalTransaction(databaseFunction)
-    })
+  protected def runInTransaction[A](dbQueryName: String, samRequestContext: SamRequestContext)(databaseFunction: DBSession => A): IO[A] = {
```

Comment:
let's add a comment about what dbQueryName and samRequestContext values should be.